spec/features/18f_procurement_spec.rb

```diff
@@ -81,94 +82,100 @@
         click_on "Submit for approval"
       }.to_not change { Cart.count }
 
-      expect(current_path).to eq('/gsa18f/proposals')
+      expect(current_path).to eq('/gsa18f/procurements')
       expect(page).to have_content("Quantity must be greater than or equal to 1")
       # keeps the form values
     end
+    let(:gsa18f) {
+      pr = FactoryGirl.create(:gsa18f_procurement)
+      pr.init_and_save_cart('approver@email.com', requester)
+      pr
+    }
+    let(:gsa18f_cart) {
+      gsa18f.proposal.cart
+    }
```

Comment:
Rather than having `let` statements in the middle of a spec file, mind moving them to the top (if they're used by all specs in the file), or surrounding this and the subsequent specs with a `context` block?